src/cf-layout/src/cf-layout.less

```diff
@@ -15,12 +15,12 @@
 // Color variables
 
 // .block
+@block__bg:                     lighten(#3a8899, 55%);
 @block__border-top:             #3a8899;
 @block__border-right:           #3a8899;
 @block__border-bottom:          #3a8899;
 @block__border-left:            #3a8899;
 @block__border:                 #3a8899;
```

Comment:
Should `@block__border` be moved up as well? Then each side can reference it so if it changes, it's only changing once.